components/DiscordIcon/DiscordIcon.tsx

```diff
@@ -1,22 +1,27 @@
 import Image from "next/image"
-import { useMediaQuery } from "usehooks-ts"
+import { useTheme } from "../../providers/ThemeProvider"
 
 const DiscordIcon = () => {
-  const isMobile = useMediaQuery("(max-width: 768px)")
+  const { themeMode } = useTheme()
 
   return (
     <a
-      href="https://discord.gg/ZpZBHCrqHQ"
+      href="https://discord.com/channels/1055217047908384849/1055218028012392519"
```

Comment:
```suggestion
      href="https://discord.gg/ZpZBHCrqHQ"
```

please do not change the Discord link. This is important for new members of our NFT project to join our community.